src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -108,15 +113,17 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
         file={file}
       >
         <form onSubmit={handleDocUpload} autoComplete="off">
-          <FormControlLabel
-            control={<Switch />}
-            label="Verify file on upload"
-            id="verify-checkbox"
-            value={verifyFile}
-            checked={verifyFile}
-            onChange={() => setVerifyFile(!verifyFile)}
-            sx={{ mb: 1 }}
-          />
+          {user !== 'personal' && (
```

Comment:
I don't think this is needed. We haven't worked out who will have authority to verify documents yet. There is a workflow that may allow for self-signing. Let's remove this for now.

Reply:
I could do that. I'll have this reverted back.